tcf_website/templates/reviews/review_form_content.html

```diff
@@ -39,9 +39,17 @@ <h2 class="h4 review-form-header form-group col-12 mt-4 mb-2">
   <h2 class="h4 review-form-header form-group col-12 my-4">
     Tell us more about the course and instructor</h2>
   <div class="form-group col mb-3">
-    <textarea class="form-control" id="text" name="text" rows="5"
+    <textarea class="form-control" id="reviewtext" name="text" rows="5"
               placeholder="(Aim for 3 sentences or more!)" minlength="100" maxlength="5000">{{ form.text.value }}</textarea>
   </div>
+    <!-- Message for Progress Bar-->
+  <div class="form-group col-12">
+    <p class="medium" id="progressbarmessage"></p>
+  </div>
+        <!-- Progress Bar -->
+  <div class="progress" style="height: 25px;">

```

Comment:
We usually try to keep styling in the css files!

btw I think it'd look nicer if the progress bar was a little thinner (15px?) and there was less spacing between the prompt text and the bar.

Reply:
Yes, thanks for pointing that out it's now 15 px.